apollo-router/src/plugins/telemetry/apollo.rs

```diff
@@ -182,6 +186,17 @@ impl Default for Config {
     }
 }
 
+#[derive(Clone, Debug, Deserialize, JsonSchema, Copy)]
+#[serde(deny_unknown_fields, rename_all = "snake_case")]
+pub(crate) enum ApolloTracingProtocol {
+    // Use only the Apollo usage reporting protobuf over http
+    Apollo,
+    // Use both the Apollo usage reporting protobuf AND Apollo OTLP
+    ApolloAndOtlp,
```

Comment:
Minor but there's a pretty common pattern here of using New/Legacy/Both as the enum names (though maybe we should use Otlp/Apollo/Both instead?) https://docs.google.com/document/d/1Zk6PJyCwgDIMTRBSt3UoCKPfefM25saKCDn_AtEHQnc/edit#heading=h.am5dyhwon4gw

Reply:
Ah cool - happy to use a convention if there is one established 👍

Reply:
Also could you change `testing` by `experimental` to be consistent with what we currently have